2025-05-07 13:55:18,616 - INFO - Создание таблиц в базе данных
2025-05-07 13:55:18,625 - INFO - Start polling
2025-05-07 13:55:21,793 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 13:56:10,897 - INFO - Обработка пересланного сообщения
2025-05-07 13:56:10,900 - INFO - Обработка пересланного сообщения
2025-05-07 13:56:10,902 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 10:56:10.900594')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 13:56:10,904 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 10:56:10.902617')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 13:56:11,102 - INFO - Update id=752632529 is handled. Duration 206 ms by bot id=7676896339
2025-05-07 13:56:11,131 - INFO - Update id=752632530 is handled. Duration 235 ms by bot id=7676896339
2025-05-07 13:58:32,086 - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2025-05-07 13:58:32,086 - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7676896339)
2025-05-07 13:58:43,929 - INFO - Connection established (tryings = 1, bot id = 7676896339)
2025-05-07 13:59:20,501 - WARNING - Received SIGINT signal
2025-05-07 13:59:20,502 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 13:59:20,502 - INFO - Polling stopped
2025-05-07 13:59:30,052 - INFO - Создание таблиц в базе данных
2025-05-07 13:59:30,065 - INFO - Start polling
2025-05-07 13:59:31,105 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 13:59:53,713 - INFO - Обработка пересланного сообщения
2025-05-07 13:59:53,716 - INFO - Обработка пересланного сообщения
2025-05-07 13:59:53,717 - INFO - Обработка пересланного сообщения
2025-05-07 13:59:53,718 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 10:59:53.716437')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 13:59:53,721 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 10:59:53.719267')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 13:59:53,722 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 10:59:53.719127')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 13:59:54,599 - INFO - Update id=752632533 is handled. Duration 885 ms by bot id=7676896339
2025-05-07 13:59:54,629 - INFO - Update id=752632532 is handled. Duration 916 ms by bot id=7676896339
2025-05-07 13:59:54,667 - INFO - Update id=752632531 is handled. Duration 954 ms by bot id=7676896339
2025-05-07 14:04:48,561 - WARNING - Received SIGINT signal
2025-05-07 14:04:48,562 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:04:48,562 - INFO - Polling stopped
2025-05-07 14:05:02,544 - INFO - Создание таблиц в базе данных
2025-05-07 14:05:02,553 - INFO - Start polling
2025-05-07 14:05:03,861 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:05:25,430 - INFO - Обработка пересланного сообщения
2025-05-07 14:05:25,434 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 11:05:25.434002')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 14:05:26,428 - INFO - Update id=752632534 is handled. Duration 997 ms by bot id=7676896339
2025-05-07 14:09:00,801 - WARNING - Received SIGINT signal
2025-05-07 14:09:00,803 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:09:00,803 - INFO - Polling stopped
2025-05-07 14:09:06,126 - INFO - Создание таблиц в базе данных
2025-05-07 14:09:06,135 - INFO - Start polling
2025-05-07 14:09:07,523 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:09:30,807 - INFO - Обработка пересланного сообщения
2025-05-07 14:09:30,812 - INFO - Обработка пересланного сообщения
2025-05-07 14:09:30,814 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 11:09:30.811937')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 14:09:30,816 - ERROR - Ошибка при сохранении канала: (sqlite3.IntegrityError) NOT NULL constraint failed: channels.id
[SQL: INSERT INTO channels (channel_id, channel_url, timestamp) VALUES (?, ?, ?)]
[parameters: (-1001478765631, 'https://t.me/yurasumy', '2025-05-07 11:09:30.814647')]
(Background on this error at: https://sqlalche.me/e/20/gkpj)
2025-05-07 14:09:31,692 - INFO - Update id=752632535 is handled. Duration 886 ms by bot id=7676896339
2025-05-07 14:09:31,724 - INFO - Update id=752632536 is handled. Duration 917 ms by bot id=7676896339
2025-05-07 14:11:42,817 - WARNING - Received SIGINT signal
2025-05-07 14:11:42,817 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:11:42,817 - INFO - Polling stopped
2025-05-07 14:11:47,495 - INFO - Создание таблиц в базе данных
2025-05-07 14:11:47,505 - INFO - Start polling
2025-05-07 14:11:48,535 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:12:06,994 - INFO - Обработка пересланного сообщения
2025-05-07 14:12:07,004 - INFO - Канал сохранён: https://t.me/Mikle1On
2025-05-07 14:12:07,916 - INFO - Update id=752632537 is handled. Duration 921 ms by bot id=7676896339
2025-05-07 14:16:27,175 - INFO - Update id=752632538 is not handled. Duration 0 ms by bot id=7676896339
2025-05-07 14:19:02,047 - INFO - Обработка пересланного сообщения
2025-05-07 14:19:02,063 - INFO - Канал сохранён: https://t.me/my_thinks
2025-05-07 14:19:02,459 - INFO - Update id=752632539 is handled. Duration 413 ms by bot id=7676896339
2025-05-07 14:19:53,396 - INFO - Обработка пересланного сообщения
2025-05-07 14:19:53,401 - INFO - Канал сохранён: https://t.me/kukuffka
2025-05-07 14:19:53,722 - INFO - Update id=752632540 is handled. Duration 326 ms by bot id=7676896339
2025-05-07 14:27:58,918 - INFO - Обработка пересланного сообщения
2025-05-07 14:27:58,925 - INFO - Канал сохранён: https://t.me/yurasumy
2025-05-07 14:27:59,243 - INFO - Update id=752632541 is handled. Duration 325 ms by bot id=7676896339
2025-05-07 14:29:04,831 - INFO - Запрос списка каналов
2025-05-07 14:29:04,835 - INFO - Список каналов отправлен
2025-05-07 14:29:05,182 - INFO - Update id=752632542 is handled. Duration 352 ms by bot id=7676896339
2025-05-07 14:32:10,216 - INFO - Обработка пересланного сообщения
2025-05-07 14:32:10,223 - INFO - Канал сохранён: https://t.me/infantmilitario
2025-05-07 14:32:10,532 - INFO - Update id=752632543 is handled. Duration 316 ms by bot id=7676896339
2025-05-07 14:38:40,002 - INFO - Обработка пересланного сообщения
2025-05-07 14:38:40,008 - INFO - Канал сохранён: https://t.me/Mikle1On
2025-05-07 14:38:40,397 - INFO - Update id=752632544 is handled. Duration 395 ms by bot id=7676896339
2025-05-07 14:38:52,540 - INFO - Запрос списка каналов
2025-05-07 14:38:52,543 - INFO - Список каналов отправлен
2025-05-07 14:38:52,853 - INFO - Update id=752632545 is handled. Duration 312 ms by bot id=7676896339
2025-05-07 14:46:04,904 - WARNING - Received SIGINT signal
2025-05-07 14:46:04,904 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:46:04,904 - INFO - Polling stopped
2025-05-07 14:46:10,535 - INFO - Создание таблиц в базе данных
2025-05-07 14:46:10,539 - INFO - Start polling
2025-05-07 14:46:11,575 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 14:46:21,833 - INFO - Запрос списка каналов
2025-05-07 14:46:21,837 - INFO - Список каналов отправлен
2025-05-07 14:46:22,769 - INFO - Update id=752632546 is handled. Duration 936 ms by bot id=7676896339
2025-05-07 14:49:36,304 - INFO - Update id=752632547 is not handled. Duration 0 ms by bot id=7676896339
2025-05-07 15:01:15,097 - WARNING - Received SIGINT signal
2025-05-07 15:01:15,098 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:01:15,098 - INFO - Polling stopped
2025-05-07 15:01:28,198 - INFO - Создание таблиц в базе данных
2025-05-07 15:01:28,201 - INFO - Start polling
2025-05-07 15:01:29,245 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:01:44,672 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:01:44,673 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:01:44,673 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-07 15:01:44,673 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:01:44,674 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:01:44,674 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-07 15:01:45,550 - INFO - Update id=752632549 is handled. Duration 878 ms by bot id=7676896339
2025-05-07 15:01:45,573 - INFO - Update id=752632548 is handled. Duration 901 ms by bot id=7676896339
2025-05-07 15:05:31,538 - WARNING - Received SIGINT signal
2025-05-07 15:05:31,539 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:05:31,539 - INFO - Polling stopped
2025-05-07 15:05:36,742 - INFO - Создание таблиц в базе данных
2025-05-07 15:05:36,745 - INFO - Start polling
2025-05-07 15:05:37,781 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:06:14,720 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:06:14,720 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:06:14,729 - INFO - Канал сохранён: https://t.me/medvedevvesti/20834
2025-05-07 15:06:15,618 - INFO - Update id=752632550 is handled. Duration 898 ms by bot id=7676896339
2025-05-07 15:06:42,657 - INFO - Запрос списка каналов
2025-05-07 15:06:42,660 - INFO - Список каналов отправлен
2025-05-07 15:06:43,050 - INFO - Update id=752632551 is handled. Duration 394 ms by bot id=7676896339
2025-05-07 15:15:40,060 - WARNING - Received SIGINT signal
2025-05-07 15:15:40,061 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:15:40,061 - INFO - Polling stopped
2025-05-07 15:15:46,107 - INFO - Создание таблиц в базе данных
2025-05-07 15:15:46,114 - INFO - Start polling
2025-05-07 15:15:47,424 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:16:17,119 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:16:17,119 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:16:17,119 - INFO - Update id=752632552 is not handled. Duration 0 ms by bot id=7676896339
2025-05-07 15:16:17,119 - ERROR - Cause exception while process update id=752632552 by bot id=7676896339
UnboundLocalError: cannot access local variable 'channel_url' where it is not associated with a value
Traceback (most recent call last):
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/oleg/Документы/Личные/Обучение/channels_collector/main.py", line 110, in handle_forwarded_from_bot
    if channel_url.count("/") >= 4:
       ^^^^^^^^^^^
UnboundLocalError: cannot access local variable 'channel_url' where it is not associated with a value
2025-05-07 15:18:11,891 - WARNING - Received SIGINT signal
2025-05-07 15:18:11,892 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:18:11,892 - INFO - Polling stopped
2025-05-07 15:18:29,294 - INFO - Создание таблиц в базе данных
2025-05-07 15:18:29,298 - INFO - Start polling
2025-05-07 15:18:30,332 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:18:40,608 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:18:40,609 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:18:40,617 - INFO - Канал сохранён: https://t.me/vzglyad_ru
2025-05-07 15:18:41,509 - INFO - Update id=752632553 is handled. Duration 901 ms by bot id=7676896339
2025-05-07 15:19:22,931 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:19:22,932 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:19:22,932 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-07 15:19:23,261 - INFO - Update id=752632554 is handled. Duration 330 ms by bot id=7676896339
2025-05-07 15:39:02,877 - WARNING - Received SIGINT signal
2025-05-07 15:39:02,878 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:39:02,878 - INFO - Polling stopped
2025-05-07 15:39:09,489 - INFO - Создание таблиц в базе данных
2025-05-07 15:39:09,493 - INFO - Start polling
2025-05-07 15:39:10,729 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:39:22,019 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:39:22,020 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:39:22,020 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-07 15:39:22,892 - INFO - Update id=752632555 is handled. Duration 873 ms by bot id=7676896339
2025-05-07 15:45:51,081 - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2025-05-07 15:45:51,081 - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7676896339)
2025-05-07 15:46:02,227 - INFO - Connection established (tryings = 1, bot id = 7676896339)
2025-05-07 15:52:38,009 - WARNING - Received SIGINT signal
2025-05-07 15:52:38,010 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:52:38,010 - INFO - Polling stopped
2025-05-07 15:52:43,610 - INFO - Создание таблиц в базе данных
2025-05-07 15:52:43,613 - INFO - Start polling
2025-05-07 15:52:43,809 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-07 15:52:50,330 - INFO - Обработка пересланного сообщения от другого бота
2025-05-07 15:52:50,330 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-07 15:52:50,330 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-07 15:52:50,528 - INFO - Update id=752632556 is handled. Duration 198 ms by bot id=7676896339
2025-05-09 18:21:02,782 - INFO - Создание таблиц в базе данных
2025-05-09 18:21:02,793 - INFO - Start polling
2025-05-09 18:21:02,985 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 18:22:45,770 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 18:22:45,775 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 18:22:45,775 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-09 18:22:46,236 - INFO - Update id=752632575 is handled. Duration 474 ms by bot id=7676896339
2025-05-09 21:20:11,696 - INFO - Создание таблиц в базе данных
2025-05-09 21:20:11,701 - INFO - Start polling
2025-05-09 21:20:12,907 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 21:21:05,066 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:21:05,066 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:21:05,067 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:21:07,756 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-09 21:23:02,470 - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2025-05-09 21:23:02,470 - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7676896339)
2025-05-09 21:23:19,767 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-09 21:23:19,768 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-09 21:23:19,772 - ERROR - Ошибка при получении названия канала: Cannot find any entity corresponding to "https://t.me/common_sen/606"
2025-05-09 21:23:19,795 - INFO - Канал сохранён: https://t.me/common_sen/606 (Неизвестно)
2025-05-09 21:23:19,797 - ERROR - Ошибка при сохранении канала: Unable to serialize unknown type: <class 'aiogram.client.default.Default'>
2025-05-09 21:23:20,688 - INFO - Update id=752632576 is not handled. Duration 135625 ms by bot id=7676896339
2025-05-09 21:23:20,689 - ERROR - Cause exception while process update id=752632576 by bot id=7676896339
TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Unsupported start tag "class" at byte offset 52
Traceback (most recent call last):
  File "/home/oleg/MyStudy/channels_collector/main.py", line 168, in handle_forwarded_from_bot
    print(message.model_dump_json(
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/pydantic/main.py", line 519, in model_dump_json
    return self.__pydantic_serializer__.to_json(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.PydanticSerializationError: Unable to serialize unknown type: <class 'aiogram.client.default.Default'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/main.py", line 175, in handle_forwarded_from_bot
    await message.reply(f"❌ Ошибка: {str(e)}")
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/methods/base.py", line 84, in emit
    return await bot(self)
           ^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/bot.py", line 478, in __call__
    return await self.session(self, method, timeout=request_timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/session/base.py", line 254, in __call__
    return cast(TelegramType, await middleware(bot, method))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/session/aiohttp.py", line 185, in make_request
    response = self.check_response(
               ^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/session/base.py", line 120, in check_response
    raise TelegramBadRequest(method=method, message=description)
aiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Unsupported start tag "class" at byte offset 52
2025-05-09 21:23:29,839 - INFO - Connection established (tryings = 1, bot id = 7676896339)
2025-05-09 21:23:41,668 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:23:41,669 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:23:41,672 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:23:42,006 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-09 21:23:42,620 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-09 21:23:42,621 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-09 21:23:42,631 - ERROR - Ошибка при получении названия канала: Cannot find any entity corresponding to "https://t.me/common_sen/606"
2025-05-09 21:23:42,660 - INFO - Канал сохранён: https://t.me/common_sen/606 (Неизвестно)
2025-05-09 21:23:42,660 - ERROR - Ошибка при сохранении канала: Unable to serialize unknown type: <class 'aiogram.client.default.Default'>
2025-05-09 21:23:42,965 - INFO - Update id=752632577 is not handled. Duration 1300 ms by bot id=7676896339
2025-05-09 21:23:42,965 - ERROR - Cause exception while process update id=752632577 by bot id=7676896339
TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Unsupported start tag "class" at byte offset 52
Traceback (most recent call last):
  File "/home/oleg/MyStudy/channels_collector/main.py", line 168, in handle_forwarded_from_bot
    print(message.model_dump_json(
          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/pydantic/main.py", line 519, in model_dump_json
    return self.__pydantic_serializer__.to_json(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
pydantic_core._pydantic_core.PydanticSerializationError: Unable to serialize unknown type: <class 'aiogram.client.default.Default'>

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 309, in _process_update
    response = await self.feed_update(bot, update, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 158, in feed_update
    response = await self.update.wrap_outer_middleware(
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/error.py", line 25, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/middlewares/user_context.py", line 56, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/fsm/middleware.py", line 42, in __call__
    return await handler(event, data)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/dispatcher.py", line 276, in _listen_update
    return await self.propagate_event(update_type=update_type, event=event, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 146, in propagate_event
    return await observer.wrap_outer_middleware(_wrapped, event=event, data=kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 141, in _wrapped
    return await self._propagate_event(
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/router.py", line 166, in _propagate_event
    response = await observer.trigger(event, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/telegram.py", line 121, in trigger
    return await wrapped_inner(event, kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/dispatcher/event/handler.py", line 43, in call
    return await wrapped()
           ^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/main.py", line 175, in handle_forwarded_from_bot
    await message.reply(f"❌ Ошибка: {str(e)}")
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/methods/base.py", line 84, in emit
    return await bot(self)
           ^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/bot.py", line 478, in __call__
    return await self.session(self, method, timeout=request_timeout)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/session/base.py", line 254, in __call__
    return cast(TelegramType, await middleware(bot, method))
                              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/session/aiohttp.py", line 185, in make_request
    response = self.check_response(
               ^^^^^^^^^^^^^^^^^^^^
  File "/home/oleg/MyStudy/channels_collector/.venv/lib/python3.12/site-packages/aiogram/client/session/base.py", line 120, in check_response
    raise TelegramBadRequest(method=method, message=description)
aiogram.exceptions.TelegramBadRequest: Telegram server says - Bad Request: can't parse entities: Unsupported start tag "class" at byte offset 52
2025-05-09 21:24:07,111 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:24:07,111 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:24:07,112 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:24:07,410 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-09 21:24:08,024 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-09 21:24:08,025 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-09 21:24:08,032 - ERROR - Ошибка при получении названия канала: Cannot find any entity corresponding to "https://t.me/vzglyad_ru/120470"
2025-05-09 21:24:08,047 - INFO - Канал сохранён: https://t.me/vzglyad_ru/120470 (Неизвестно)
2025-05-09 21:24:08,388 - INFO - Update id=752632578 is handled. Duration 1279 ms by bot id=7676896339
2025-05-09 21:24:34,108 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:24:34,109 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:24:34,109 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:24:34,510 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-09 21:24:35,175 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-09 21:24:35,177 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-09 21:24:35,188 - ERROR - Ошибка при получении названия канала: Cannot find any entity corresponding to "https://t.me/voenkorkotenok/64125"
2025-05-09 21:24:35,216 - INFO - Канал сохранён: https://t.me/voenkorkotenok/64125 (Неизвестно)
2025-05-09 21:24:35,821 - INFO - Update id=752632579 is handled. Duration 1713 ms by bot id=7676896339
2025-05-09 21:24:53,266 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:24:53,266 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:24:53,266 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:24:53,550 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-09 21:24:54,181 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-09 21:24:54,183 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-09 21:24:54,190 - ERROR - Ошибка при получении названия канала: Cannot find any entity corresponding to "https://t.me/readovkanews/96145"
2025-05-09 21:24:54,225 - INFO - Канал сохранён: https://t.me/readovkanews/96145 (Неизвестно)
2025-05-09 21:24:55,006 - INFO - Update id=752632580 is handled. Duration 1742 ms by bot id=7676896339
2025-05-09 21:25:33,922 - INFO - Запрос списка названий каналов
2025-05-09 21:25:33,928 - INFO - Список названий каналов отправлен
2025-05-09 21:25:34,344 - INFO - Update id=752632581 is handled. Duration 422 ms by bot id=7676896339
2025-05-09 21:31:00,790 - WARNING - Received SIGINT signal
2025-05-09 21:31:00,793 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 21:31:00,793 - INFO - Polling stopped
2025-05-09 21:31:10,339 - INFO - Создание таблиц в базе данных
2025-05-09 21:31:10,344 - INFO - Start polling
2025-05-09 21:31:11,527 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 21:32:10,620 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:32:10,620 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:32:10,621 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:32:10,903 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-09 21:35:24,767 - ERROR - Ошибка при получении названия канала: 3 consecutive sign-in attempts failed. Aborting
2025-05-09 21:35:24,792 - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2025-05-09 21:35:24,792 - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7676896339)
2025-05-09 21:35:24,793 - WARNING - Received SIGINT signal
2025-05-09 21:35:24,794 - WARNING - Received SIGINT signal
2025-05-09 21:35:24,794 - WARNING - Received SIGINT signal
2025-05-09 21:35:24,802 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 21:35:24,802 - INFO - Polling stopped
2025-05-09 21:35:24,818 - INFO - Канал сохранён: https://t.me/+MA0k1Y-VxRI0MTU6 (Неизвестно)
2025-05-09 21:35:25,055 - INFO - Update id=752632582 is not handled. Duration 194438 ms by bot id=7676896339
2025-05-09 21:35:33,775 - INFO - Создание таблиц в базе данных
2025-05-09 21:35:33,780 - INFO - Start polling
2025-05-09 21:35:34,716 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 21:35:44,966 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:35:44,966 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:35:44,967 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:35:45,250 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-09 21:38:25,810 - ERROR - Ошибка при получении названия канала: 3 consecutive sign-in attempts failed. Aborting
2025-05-09 21:38:25,828 - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2025-05-09 21:38:25,828 - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7676896339)
2025-05-09 21:38:25,828 - WARNING - Received SIGINT signal
2025-05-09 21:38:25,828 - WARNING - Received SIGINT signal
2025-05-09 21:38:25,829 - WARNING - Received SIGINT signal
2025-05-09 21:38:25,834 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 21:38:25,834 - INFO - Polling stopped
2025-05-09 21:38:25,855 - INFO - Канал сохранён: https://t.me/+MA0k1Y-VxRI0MTU6 (Неизвестно)
2025-05-09 21:38:26,087 - INFO - Update id=752632583 is not handled. Duration 161122 ms by bot id=7676896339
2025-05-09 21:38:34,737 - INFO - Создание таблиц в базе данных
2025-05-09 21:38:34,744 - INFO - Start polling
2025-05-09 21:38:35,639 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-09 21:38:39,480 - INFO - Обработка пересланного сообщения от другого бота
2025-05-09 21:38:39,481 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-09 21:38:39,482 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-09 21:38:39,761 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-10 12:01:07,330 - ERROR - Ошибка при получении названия канала: 3 consecutive sign-in attempts failed. Aborting
2025-05-10 12:01:07,353 - ERROR - Failed to fetch updates - TelegramNetworkError: HTTP Client says - Request timeout error
2025-05-10 12:01:07,354 - WARNING - Sleep for 1.000000 seconds and try again... (tryings = 0, bot id = 7676896339)
2025-05-10 12:01:07,355 - WARNING - Received SIGINT signal
2025-05-10 12:01:07,356 - WARNING - Received SIGINT signal
2025-05-10 12:01:07,356 - WARNING - Received SIGINT signal
2025-05-10 12:01:07,364 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-10 12:01:07,365 - INFO - Polling stopped
2025-05-10 12:01:07,389 - INFO - Канал сохранён: https://t.me/+MA0k1Y-VxRI0MTU6 (Неизвестно)
2025-05-10 12:01:07,619 - INFO - Update id=752632584 is not handled. Duration 129054 ms by bot id=7676896339
2025-05-10 12:02:06,915 - INFO - Создание таблиц в базе данных
2025-05-10 12:02:06,919 - INFO - Start polling
2025-05-10 12:02:07,105 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-10 12:02:30,138 - INFO - Обработка пересланного сообщения от другого бота
2025-05-10 12:02:30,139 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-10 12:02:30,140 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-10 12:02:30,193 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-10 12:03:03,593 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-10 12:03:03,595 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-10 12:03:03,632 - INFO - Канал сохранён: https://t.me/boris_rozhin/164171 (Colonelcassad)
2025-05-10 12:03:04,015 - INFO - Update id=752632585 is handled. Duration 33880 ms by bot id=7676896339
2025-05-10 12:03:51,838 - INFO - Запрос списка названий каналов
2025-05-10 12:03:51,844 - INFO - Список названий каналов отправлен
2025-05-10 12:03:51,933 - INFO - Update id=752632586 is handled. Duration 96 ms by bot id=7676896339
2025-05-10 12:04:17,188 - INFO - Обработка пересланного сообщения от другого бота
2025-05-10 12:04:17,188 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-10 12:04:17,189 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-10 12:04:17,190 - INFO - Обработка пересланного сообщения от другого бота
2025-05-10 12:04:17,191 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-10 12:04:17,192 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-10 12:04:17,241 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-10 12:04:17,271 - INFO - Update id=752632588 is handled. Duration 86 ms by bot id=7676896339
2025-05-10 12:04:17,444 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-10 12:04:17,446 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-10 12:04:17,479 - INFO - Канал сохранён: https://t.me/yuzhny_front_ZOV (Южный фронт)
2025-05-10 12:04:17,561 - INFO - Update id=752632587 is handled. Duration 378 ms by bot id=7676896339
2025-05-10 12:11:30,656 - WARNING - Received SIGINT signal
2025-05-10 12:11:30,659 - INFO - Polling stopped for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-10 12:11:30,659 - INFO - Polling stopped
2025-05-10 12:13:30,462 - INFO - Создание таблиц в базе данных
2025-05-10 12:13:30,466 - INFO - Start polling
2025-05-10 12:13:30,645 - INFO - Run polling for bot @channels_collector_bot id=7676896339 - 'Мой сборщик каналов'
2025-05-10 12:13:42,366 - INFO - Обработка пересланного сообщения от другого бота
2025-05-10 12:13:42,367 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-10 12:13:42,367 - INFO - Connecting to 149.154.167.51:443/TcpFull...
2025-05-10 12:13:42,370 - INFO - Обработка пересланного сообщения от другого бота
2025-05-10 12:13:42,370 - INFO - Сообщение переслано от бота: junction_bot (ID: 374139448)
2025-05-10 12:13:42,370 - WARNING - Не удалось извлечь ссылку на канал из сообщения
2025-05-10 12:13:42,419 - INFO - Connection to 149.154.167.51:443/TcpFull complete!
2025-05-10 12:13:42,655 - INFO - Update id=752632590 is handled. Duration 292 ms by bot id=7676896339
2025-05-10 12:13:42,886 - INFO - Disconnecting from 149.154.167.51:443/TcpFull...
2025-05-10 12:13:42,886 - INFO - Disconnection from 149.154.167.51:443/TcpFull complete!
2025-05-10 12:13:42,930 - INFO - Канал сохранён: https://t.me/rvvoenkor/91430 (Операция Z: Военкоры Русской Весны)
2025-05-10 12:13:43,047 - INFO - Update id=752632589 is handled. Duration 685 ms by bot id=7676896339
2025-05-10 12:18:51,607 - INFO - Обработка пересланного сообщения
2025-05-10 12:18:51,616 - INFO - Канал сохранён: https://t.me/Mikle1On (Мир Михаила Онуфриенко)
2025-05-10 12:18:51,698 - INFO - Update id=752632591 is handled. Duration 92 ms by bot id=7676896339
